samples/wifi/scan/prj.conf

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_DNS_RESOLVER=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=153600
+
+CONFIG_INIT_STACKS=y
+
+CONFIG_NET_L2_ETHERNET=y
+
+# Memories
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_NET_TX_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=4096
+
+# Debugging
+CONFIG_STACK_SENTINEL=y
+CONFIG_DEBUG_COREDUMP=y
+CONFIG_DEBUG_COREDUMP_BACKEND_LOGGING=y
+CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
+CONFIG_SHELL_CMDS_RESIZE=n
+
+# Logging
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
```

Comment:
@krish2718  Few scan-results got truncated by using LOG_INF() and also SSID field is updating empty. So, to fix that, added LOG_MODE_MINIMAL config.

Reply:
Ah, we need to increase the LOG buffer size, but as the original wifi_shell, lets use `printk` itself.

Reply:
@krish2718 Replaced logger with printk.